From: Michal Malek <michalm@jabster.pl>
Date: Sun, 03 Jul 2011 21:55:16 +0000
Subject: Fixed improper track number in CDDB track edit window title
X-Git-Url: http://quickgit.kde.org/?p=k3b.git&a=commitdiff&h=6110dc547b118a6aecc64f15bbcef04636854958
---
Fixed improper track number in CDDB track edit window title

The title showed track index instead of a track number (which equals to
track index + 1).

BUG: 276681
FIXED-IN: 2.0.3
---


--- a/src/rip/k3baudiocdview.cpp
+++ b/src/rip/k3baudiocdview.cpp
@@ -324,7 +324,7 @@
         int trackIndex = items.first();
 
         KDialog dialog( this);
-        dialog.setCaption(i18n("CDDB Track %1", trackIndex) );
+        dialog.setCaption(i18n("CDDB Track %1", trackIndex+1) );
         dialog.setButtons(KDialog::Ok|KDialog::Cancel);
         dialog.setDefaultButton(KDialog::Ok);
         dialog.setModal(true);

